src/assets/metadata/Balancer.json

```diff
@@ -0,0 +1,5 @@
+{
+  "version": "0.1.0",
+  "appCode": "Balancer",
```

Comment:
I don't think we should include the balance file in the cowswap UI

Reply:
okokko, was just to make it easy for us, but a document would be better suited. Removing. Created also: https://docs.google.com/spreadsheets/d/16RzefdfpmEavl182h6iNEe7fRyAcaATCRYR0hMQ_734/edit#gid=0